runtime/src/snapshot_utils.rs

```diff
@@ -417,9 +543,9 @@ pub fn archive_snapshot_package(
     Ok(())
 }
 
-pub fn get_snapshot_paths<P: AsRef<Path>>(snapshot_path: P) -> Vec<SlotSnapshotPaths>
+pub fn get_snapshot_paths<P>(snapshot_path: P) -> Vec<SlotSnapshotPaths>
```

Comment:
nit: This is getting the storage entry paths right? How about `get_slot_storage_paths`